lib/audit_test.pm

```diff
@@ -121,4 +123,47 @@ sub compare_run_log {
     return $result;
 }
 
+sub lxj_parse_lines {
+    my ($lines) = @_;
+    my $results = {};
+    foreach my $line (@$lines) {
+        if ($line =~ /(\[\d+\])\s+(\S+)\s+(PASS|FAIL|ERROR)/) {
+            $results->{$2} = $3;
+        }
+    }
+    return $results;
+}
+
+sub lxj_compare_run_log {
+    my ($current_results, $baseline) = @_;
+    my $baseline_results = {};
+    my $baseline_file    = "ulogs/$baseline";
+    if (!-e $baseline_file) {
+        diag "The file $baseline_file does not exist";
+    }
+    else {
+        my @lines = split(/\n/, path("$baseline_file")->slurp);
+        $baseline_results = lxj_parse_lines(\@lines);
+    }
+    my $results = {};
+    my $flag    = 'ok';
+    foreach my $c_key (keys %$current_results) {
+        my $current_result = $current_results->{$c_key};
+        unless ($baseline_results->{$c_key}) {
```

Comment:
In this situation should test case send out a warning/softfailure to test user for updating the baseline.